Documentation/5.4.1/README.rst

```diff
@@ -418,116 +425,253 @@ This example shows how to create a deployment with one Controller node and two C
+                      get_param: BondInterfaceOvsOptions
+                    members:
+                    - type: sriov_pf
+                      name:
+                        get_param: MellanoxTenantPort1
+                      link_mode: switchdev
+                      numvfs: 8
+                      promisc: true
+                      use_dhcp: false
+                      primary: true
+                    - type: sriov_pf
+                      name:
+                        get_param: MellanoxTenantPort2
+                      link_mode: switchdev
+                      numvfs: 8
+                      promisc: true
+                      use_dhcp: false
+                  - type: vlan
+                    device: tenant-bond
+                    vlan_id:
+                      get_param: TenantNetworkVlanID
+                    addresses:
+                    - ip_netmask:
+                        get_param: TenantIpSubnet
+                ...
 
-::
 
-    OvercloudControllerFlavor: control
-    ControllerCount: 1
-    OvercloudComputeFlavor: compute
-    ComputeCount: 2
-    OvercloudComputeSriovFlavor: computesriov
-    ComputeAvrsCount: 2
+:Step 8: For "Deploy Overcloud", we need to pass ``/usr/share/openstack-tripleo-heat-templates/environments/host-config-and-reboot.yaml`` as environment file.
 
 
-:Step 8: To deploy the Overcloud, additional parameters and template files are required.
+:Step 9: We also need to create ``/home/stack/templates/ovs-hw-offload.yaml`` and ``/home/stack/nuage-tripleo-heat-templates/environments/mellanox-environment.yaml`` environment files.
+
+
+:Step 10: There are no changes required for ``/usr/share/openstack-tripleo-heat-templates/environments/host-config-and-reboot.yaml``. We need set some parameters in ``/home/stack/templates/ovs-hw-offload.yaml`` and ``/home/stack/nuage-tripleo-heat-templates/environments/mellanox-environment.yaml``. A sample file is provided in `Sample Templates`_ section.
 
-    * Include the following parameter values in the heat template neutron-nuage-config.yaml:
-    
-    ::
-    
-         NeutronServicePlugins: 'NuagePortAttributes,NuageAPI,NuageL3,trunk,NuageNetTopology'
-         NeutronTypeDrivers: "vlan,vxlan,flat"
-         NeutronMechanismDrivers: ['nuage','nuage_sriov','sriovnicswitch']
-         NeutronFlatNetworks: '*'
-         NeutronTunnelIdRanges: "1:1000"
-         NeutronNetworkVLANRanges: "physnet1:2:100,physnet2:2:100"
-         NeutronVniRanges: "1001:2000"
-    
-    
-    * Add this parameter value in the heat template nova-nuage-config.yaml:
-    
-    ::
-    
-        NovaPCIPassthrough: "[{"devname":"eno2","physical_network":"physnet1"},{"devname":"eno3","physical_network":"physnet2"}]"
-    
-    
-    * Include "neutron-sriov.yaml" file in the Overcloud deployment command. See the sample in the `Sample Templates`_ section.
 
 
 5. **(Optional)** To enable Linux bonding with VLANs, perform the following instructions:
```

Comment:
Move all network related template changes to a single section.

Reply:
done